server/src/main/java/org/elasticsearch/action/admin/indices/settings/get/TransportGetSettingsAction.java

```diff
@@ -73,21 +77,27 @@ protected GetSettingsResponse newResponse() {
     protected void masterOperation(GetSettingsRequest request, ClusterState state, ActionListener<GetSettingsResponse> listener) {
         Index[] concreteIndices = indexNameExpressionResolver.concreteIndices(state, request);
         ImmutableOpenMap.Builder<String, Settings> indexToSettingsBuilder = ImmutableOpenMap.builder();
+        ImmutableOpenMap.Builder<String, Settings> indexToDefaultSettingsBuilder = ImmutableOpenMap.builder();
         for (Index concreteIndex : concreteIndices) {
             IndexMetaData indexMetaData = state.getMetaData().index(concreteIndex);
             if (indexMetaData == null) {
                 continue;
             }
 
-            Settings settings = settingsFilter.filter(indexMetaData.getSettings());
+            Settings indexSettings = settingsFilter.filter(indexMetaData.getSettings());
```

Comment:
nit: I am fine this rename, but doing it as part of this PR makes reviewing a bit trickier, it would be easier to spot what changed without it.

Reply:
it's ok, keep it. I have already looked at the changes and they are fine.